movo_network/movo_network_config.bash

```diff
@@ -14,23 +14,23 @@
 # A remote PC is ROS slave, the default interface is lo (change it to whatever physical interface is connected to movo if working with real hardware), 
 #    and the IP address is DHCP from MOVO1
 
-if [ "$HOSTNAME" = MOVO1 ]; then
-    export ROBOT_NETWORK=eth0
-    export ROS_IP=$(ip -4 address show $ROBOT_NETWORK | grep 'inet' | sed 's/.*inet \([0-9\.]\+\).*/\1/')
-    export ROS_MASTER_URI=http://MOVO2:11311/
-elif [ "$HOSTNAME" = MOVO2 ]; then
+if [ "$HOSTNAME" = movo1 ]; then
+    export ROBOT_NETWORK=enp0s25
```

Comment:
@alexswerner should this be the br0 interface? the change u left on the movo was only for movo2 case

Reply:
On movo1 br0 does not exist, only on movo2. So enp... is correct for movo1.